### DISCUSSION TITLE
Suroi

### DISCUSSION THREAD

**Discussion by XDOGOOFLILNOOB** [HasangerGames/suroi#147] [General]
Do Y’all really have a savannah mode

**Comment by XDOGOOFLILNOOB** [comment]
And how do I mark my own skin

**Comment by Katloo24** [comment]
> Do Y’all really have a savannah mode

The Savannah mode is a mod by some other people, join Suroi modding community to learn more


> And how do I mark my own skin

make your own skin and if it fits the skin rules and is good we may add it to the game 

@XDOGOOFLILNOOB

**Reply by XDOGOOFLILNOOB** [reply]
May u send link to suroi modding community

**Reply by Katloo24** [reply]
I don’t have the link on hand but if you join our discord server I can refer you to someone who does have link

**Reply by XDOGOOFLILNOOB** [reply]
I just going to make a circle with this face

**Reply by blazingyoda** [reply]
@XDOGOOFLILNOOB https://discord.gg/vauK4rAqVT

**Comment by XDOGOOFLILNOOB** [comment]
And may u make this face a suroi emote

**Reply by Katloo24** [reply]
What face

**Reply by XDOGOOFLILNOOB** [reply]
The face on by my name
![IMG_0704](https://github.com/HasangerGames/suroi/assets/73836032/d98578f0-f1e4-489d-adf0-35d962311c7b)

**Reply by Katloo24** [reply]
I don’t think we will add that

**Reply by XDOGOOFLILNOOB** [reply]
Ah

**Reply by XDOGOOFLILNOOB** [reply]
Who’s Monet

**Reply by XDOGOOFLILNOOB** [reply]
And why does it seem like she is hacking

**Comment by XDOGOOFLILNOOB** [comment]
I was banned for hacking?

**Reply by Katloo24** [reply]
Talk to us on the discord 
https://discord.gg/suroi-1077043833621184563

**Reply by XDOGOOFLILNOOB** [reply]
K

**Reply by XDOGOOFLILNOOB** [reply]
Can’t my college  has blocked discord

**Reply by Katloo24** [reply]
Join on your phone or use a vpn

**Reply by XDOGOOFLILNOOB** [reply]
I was banned

**Reply by XDOGOOFLILNOOB** [reply]
For speedrun
